homework/week-0/pooja/src/nyc/c4q/poojawins/Weapon.java

```diff
@@ -0,0 +1,22 @@
+package nyc.c4q.poojawins;
+
+/**
+ * Created by pooja on 5/1/15.
+ */
+public class Weapon extends Item {
+    private int damage;
+
+    public Weapon(String name, String description, int damage ) {
+        super(name, description);
+        this.damage = damage;
+    }
+
+    public int getDamage() {
+        return damage;
+    }
+
+    public void use(Weapon weapon) {
```

Comment:
??

I think your code is largely self-documenting and that's awesome, but because this assignment is unfinished it would benefit from comments to explain where you're at or what you intend to do with your code. I think your code would benefit from more comments in general.